sdk/recoveryservices-backup/Microsoft.Azure.Management.RecoveryServices.Backup/src/recoveryservicesbackup/Generated/Models/AzureIaaSClassicComputeVMProtectedItem.cs

```diff
@@ -96,8 +99,8 @@ public AzureIaaSClassicComputeVMProtectedItem()
         /// item.</param>
         /// <param name="extendedInfo">Additional information for this backup
         /// item.</param>
-        public AzureIaaSClassicComputeVMProtectedItem(string backupManagementType = default(string), string workloadType = default(string), string containerName = default(string), string sourceResourceId = default(string), string policyId = default(string), System.DateTime? lastRecoveryPoint = default(System.DateTime?), string backupSetName = default(string), string createMode = default(string), System.DateTime? deferredDeleteTimeInUTC = default(System.DateTime?), bool? isScheduledForDeferredDelete = default(bool?), string deferredDeleteTimeRemaining = default(string), bool? isDeferredDeleteScheduleUpcoming = default(bool?), bool? isRehydrate = default(bool?), IList<string> resourceGuardOperationRequests = default(IList<string>), bool? isArchiveEnabled = default(bool?), string policyName = default(string), string friendlyName = default(string), string virtualMachineId = default(string), string protectionStatus = default(string), string protectionState = default(string), string healthStatus = default(string), IList<AzureIaaSVMHealthDetails> healthDetails = default(IList<AzureIaaSVMHealthDetails>), IDictionary<string, KPIResourceHealthDetails> kpisHealths = default(IDictionary<string, KPIResourceHealthDetails>), string lastBackupStatus = default(string), System.DateTime? lastBackupTime = default(System.DateTime?), string protectedItemDataId = default(string), AzureIaaSVMProtectedItemExtendedInfo extendedInfo = default(AzureIaaSVMProtectedItemExtendedInfo), ExtendedProperties extendedProperties = default(ExtendedProperties))
```

Comment:
Breaking change: `softDeleteRetentionPeriod` is inserted into existing parameter lists.

Reply:
There are several similar breaking changes in the files below as well.

Reply:
Hi @archerzz , yes this is a preview release. We'd like to go ahead with the breaking changes for the time being.

Reply:
Correction: we are checking, will get back to you by tomorrow.

Reply:
Hi @archerzz we introduced the change in our swagger here: <https://github.com/Azure/azure-rest-api-specs/pull/20567>, would you know why it is coming up as a required field in the SDK when it isn't mentioned as such in the swagger itself?

Reply:
Hi @archerzz ,
We had introduced the `softdeleteRetentionPeriod` property to swagger in the PR linked: <https://github.com/Azure/azure-rest-api-specs/pull/20567>.  It is not flagged as a "required" property, neither are several other properties such as `isScheduledForDeferredDelete`. In the SDK, the non-required properties are in the constructors in this form:

```
constructor(..., type property = default(type), ...)
    : base(..., property, ...)
```

Likewise, the `softdeleteRetentionPeriod` property has been added in this way: with a default value in all public-facing constructors. Does this still count as a breaking change?

Reply:
It does appear that way. So to clarify: the modification to existing parameter lists of the base class (where the parameter list of the child class is also modified but with an optional parameter) counts as a breaking change?

Reply:
@archerzz since the updated parameters are optional and the generated SDK does not seem to be causing any issues with our Powershell (and in the case of the Python SDK) CLI clients, I think the changes should be good to go?